pkg/cmd/grafana-cli/commands/commands.go

```diff
@@ -16,23 +17,37 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
-func runDbCommand(command func(commandLine utils.CommandLine, sqlStore *sqlstore.SQLStore) error) func(context *cli.Context) error {
+// nolint: unused,deadcode
```

Comment:
I don't want to include the command itself (see https://github.com/grafana/grafana/pull/41454/files) into the same pull request to keep it as small as possible.